--- a/amd/dkms/dkms-config.mk
+++ b/amd/dkms/dkms-config.mk
@@ -0,0 +1,15 @@
+export OS_NAME=bottlerocket
+export OS_VERSION=1.0
+subdir-ccflags-y += -DOS_NAME_BOTTLEROCKET
+subdir-ccflags-y += -DOS_VERSION_MAJOR=1
+subdir-ccflags-y += -DOS_VERSION_MINOR=0
+subdir-ccflags-y += -DDRM_VER=6 -DDRM_PATCH=12 -DDRM_SUB="0"
+export CONFIG_HSA_AMD=y
+export CONFIG_DRM_AMDGPU_CIK=y
+export CONFIG_DRM_AMDGPU_SI=y
+export CONFIG_DRM_AMDGPU_USERPTR=y
+export CONFIG_DRM_AMD_DC=y
+subdir-ccflags-y += -DCONFIG_HSA_AMD
+subdir-ccflags-y += -DCONFIG_DRM_AMDGPU_CIK
+subdir-ccflags-y += -DCONFIG_DRM_AMDGPU_SI
+subdir-ccflags-y += -DCONFIG_DRM_AMDGPU_USERPTR
+subdir-ccflags-y += -DCONFIG_DRM_AMD_DC
+subdir-ccflags-y += -DCONFIG_DRM_HDCP=n
--- a/amd/dkms/config/config.h
+++ b/amd/dkms/config/config.h
@@ -0,0 +1,8 @@
+/* config.h - Static configuration for Bottlerocket */
+#define PACKAGE_NAME "amdgpu-dkms"
+#define PACKAGE_VERSION "30.10.2"
+#define HAVE_DMA_RESV_SEQ_BUG 1
+#define HAVE_RESERVATION_WW_CLASS_BUG 1
+#define HAVE_AMDKCL_FLAGS_TAKE_PATH 1
+#define HAVE_KVREALLOC_3ARG 1
+#define HAVE_AMDKCL_HMM_MIRROR_ENABLED 1
