From 37d944ce422f5716d3219440460eb780f4979dd5 Mon Sep 17 00:00:00 2001
From: Martin Harriman <mharrimn@amazon.com>
Date: Sun, 30 Mar 2025 22:36:38 +0000
Subject: [PATCH] Select prerequisites for gpu drivers

---
 drivers/gpu/drm/tiny/Kconfig | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/drivers/gpu/drm/tiny/Kconfig b/drivers/gpu/drm/tiny/Kconfig
index ce17143d47a8..6e04aba1711a 100644
--- a/drivers/gpu/drm/tiny/Kconfig
+++ b/drivers/gpu/drm/tiny/Kconfig
@@ -86,6 +86,18 @@ config DRM_SIMPLEDRM
 	select APERTURE_HELPERS
 	select DRM_GEM_SHMEM_HELPER
 	select DRM_KMS_HELPER
+	select DRM_DISPLAY_DP_HELPER
+	select DRM_DISPLAY_HELPER
+	select DRM_KMS_HELPER
+	select DRM_SUBALLOC_HELPER
+	select DRM_TTM
+	select DRM_TTM_HELPER
+	select DRM_DISPLAY_HDCP_HELPER
+	select DRM_DISPLAY_HDMI_HELPER
+	select DRM_EXEC
+	select DRM_BUDDY
+	select DRM_SCHED
+	select DRM_PANEL_BACKLIGHT_QUIRKS
 	help
 	  DRM driver for simple platform-provided framebuffers.
 
-- 
2.50.1

